fabfile.py

```diff
@@ -371,17 +372,22 @@ def after_installation(self):
         #)
 
     def send_config_to_mail(self, url=False, version=False, email=False):
-        msg = '\
-            Dear User, below are details of you newly created odoo instance:\
-            name: {0}. \
-            url: {1}. \
-            version: {2} \
-            username: admin. \
-            password: admin. \
-            You can change the password after login. \
-            Regards, \
-            Sunflower IT.'.format(self.instance, url, version)
-        sudo('echo {} | mail {}'.format(msg, email))
+        msg = "\
+            Dear User odoo-{0}, below are details of you newly created odoo instance:\n\n\
+            name: {0}.\n\
+            url: {1}.\n\
+            version: {2}\n\
+            username: admin.\n\
+            password: admin.\n\n\
+            You can change the password after login.\n\n\
+            Regards,\n\
+            Sunflower IT.".format(self.instance, url, version)
```

Comment:
I think you did not fully understand my message about the strings.

If you do it like this, you might just as well use the triple quotes:

```
msg = """
      name: {0}
      url: {1}
"""
```

It will be much cleaner.

The only reason to ever use `\n\` is when you use it like this:

```
msg = \
    "Blabla\n"\
    "Blabla\n"
```

In this case it makes sense because you are indenting your code (making the code look nicer) but you are preventing your real output from having the indendation also.

You could achieve the same with
```
       msg = """
Blabla
Blabla
       """
```

But it looks bad.